syntax = "proto3";

service ConnectionService {
  rpc getCon (conRequest) returns (conReply) {}
}

message conRequest {
  string req = 1;
}

message conReply {
  string allowed = 1;
}



service DetectorService {
    rpc detectSlave (deRequest)  returns (deReply) {}
}

message deRequest {
    string req = 1 ;
}

message deReply {
    string master_status = 1 ;
}



service TaskService {
    rpc sendTask (tRequest) returns (tReply) {}
}

message tRequest {
    string req = 1 ;
    string return_task = 2 ;
    string result_json = 3 ;
    string remark_json = 4 ;
}

message tReply {
    string rep = 1 ;
    string task_info = 2 ;
}
